r/vignettes/install.Rmd

```diff
@@ -123,6 +123,39 @@ you'll need to reinstall the package in order to enable S3 support.
 
 # How dependencies are resolved
 
+There are a number of scripts that are triggered when `R CMD INSTALL .` is run. 
+For Arrow users, these should all just work without configuration and pull in 
+the most complete pieces (e.g. official binaries that we host).
```

Comment:
I wonder if it would be good to flag that most of this information is really only intended to help developers and not day-to-day installers of Arrow? I don't mind (and actually kind of like!) that it's in the install vignette as opposed to the developing vignette, but most of the information is still probably targets at (and most relevant to) developers and not standard installers of Arrow. 

There's also a reference at the end that says "(If you're looking at this script, and you've gotten this far, it might look 
incredibly familiar: it's basically the contents of this guide in script form — 
with a few important changes)" which we probably should link over to the developing vignette there.

I don't think we want to take out some of the content just because it's not relevant to installers (since it's useful info and we don't want to have two different largely overlapping sections in both vignettes)

Reply:
Good point, will have a look at doing a better job of signposting who this is for and change up that later sentence!